docs/FAQ.rst

```diff
@@ -42,6 +42,16 @@ The TCS is defined by the Enterprise Ethereum Alliance (EEA) Task Force and
 is at
 https://entethalliance.github.io/trusted-computing/spec.html
 
+What is the difference between Proxy Mode and Direct Mode?
+----------------------------------------------------------
+Proxy Mode is where the requester (client) executes a smart contract
+(or chaincode or similar) on the blockchain.
+Th smart contract invoktes an Avalon executes an
```

Comment:
Th -> The
invoktes an -> invokes
The smart contact event invokes Avalon and Avalon executes

Reply:
Fixed.